Makefile

```diff
@@ -7,7 +7,7 @@ VERSION     := $(shell git describe --tags 2> /dev/null)
 BUILD_DATE  := $(shell date +%s)
 GIT_COMMIT_FULL  := $(shell git rev-parse HEAD)
 GO_VERSION := $(shell expr `go version |cut -d ' ' -f3 |cut -d. -f2` \>= 16)
-DOCKER_IMAGE  := "ghcr.io/metal-toolbox/conditionorc"
+DOCKER_IMAGE  ?= "ghcr.io/metal-toolbox/conditionorc"
```

Comment:
Question of curiousity: Can DOCKER_IMAGE sometimes be set outside of the makefile? Or is this just to allow that for testing purposes?